### DISCUSSION TITLE
Clarification regarding "skip_hotkeys":1

### DISCUSSION THREAD

**Discussion by Valour549** [seerge/g-helper#3210] [Q&A]
Experimental build 0.194
> Ctrl + Shift + F5 / Ctrl + Shift + Alt + F5 - **Toggle Performance Modes**
> Ctrl + Shift + F12 - **Open G-Helper window**
> Ctrl + M1 / M2 - **Screen brightness Down / Up**
> Shift + M1 / M2 - **Backlight brightness Down / Up**
> 
> If you don't want this bindings to work you can add
> "skip_hotkeys":1,

After adding the command, **Screen brightness Down / Up** and **Backlight brightness Down / Up** no longer works as expected, but **Toggle Performance Modes** and **Open G-Helper window** still work.

Further adding "keybind_profile": 0 and "keybind_app": 0 does indeed disable them, but I just want to confirm that "skip_hotkeys":1 really isn't meant to affect them?

**Comment by seerge** [comment]
@Valour549 hello, this is correct. `skip_hotkeys` does not disable `Toggle Performance Mode` or `G-Helper window`, as those keybindings have own separate config param :) I have [adjusted FAQ](https://github.com/seerge/g-helper/wiki/Power-user-settings#disable-extra-keybindings) to be more clear.
